documentation/modules/auxiliary/scanner/http/http_hsts.md

```diff
@@ -0,0 +1,296 @@
+## Vulnerable Application
+
+### Description
+
+This module checks to see whether or not the scanned systems return the HSTS header to enforce HSTS.
+
+### Install on Ubuntu
```

Comment:
which ubuntu, 18.04?

Reply:
Yeah was meant to be 18.04, will fix that up so long.